cwltool/schemas/v1.0/Process.yml

```diff
@@ -562,7 +562,9 @@ $graph:
 
     - name: default
       type: Any?
-      jsonldPredicate: "cwl:default"
+      jsonldPredicate:
```

Comment:
Oh, the `cwltool/schemas` directory is a git subtree from https://github.com/common-workflow-language/common-workflow-language/ -- can you move your changes there?